config/multerConfig.js

```diff
@@ -0,0 +1,26 @@
+const multer = require('multer');
+const path = require('path');
+
+const storage = multer.diskStorage({
+    destination: function(req, file, cb) {
```

Comment:
Widzę że masz styl pisania taki że używasz arrow function. To dobry styl i się tego warto trzymać. I jeśli coś kopujesz to warto sie zastanowić jak dostosować kod do mojego stylu. I tu można też użyć arrow function.